### ISSUE TITLE
user Roles

### ISSUE THREAD

**Issue by tyoshu** [yamcs/yamcs#838]
I am trying to create a role that will allow a user to be able to create displays via the web gui in the telemetry section. Is that controlled in ManageBucket ? Also is there any documentation of what each of the settings control? Thanks.

![Screenshot 2023-10-25 at 1 08 03 PM](https://github.com/yamcs/yamcs/assets/149006273/0cccf29a-03cb-4379-8979-de5d3ea15774)

**Comment by fqqb** [comment]
Yes, displays are stored in a bucket (by default called 'buckets'), so ManageBucket is sufficient:

```yaml
ManageBucket:
  - "displays"
```

But in your image, you already assign `ManageAnyBucket`, are you saying this is not working for you?
